Sources/StreamChatUI/CommonViews/ChatSuggestions/ChatSuggestionsCommandCollectionViewCell.swift

```diff
@@ -6,10 +6,10 @@ import StreamChat
 import UIKit
 
 /// A view that displays the command name, image and arguments.
-public typealias ChatMessageComposerCommandCellView = _ChatMessageComposerCommandCellView<NoExtraData>
+public typealias ChatSuggestionsCommandCellView = _ChatSuggestionsCommandCellView<NoExtraData>
 
 /// A view that displays the command name, image and arguments.
-open class _ChatMessageComposerCommandCellView<ExtraData: ExtraDataTypes>: _View, UIConfigProvider {
+open class _ChatSuggestionsCommandCellView<ExtraData: ExtraDataTypes>: _View, UIConfigProvider {
```

Comment:
I think we should remove this pattern of naming `CellView` suffix. It should just be `ChatSuggestionsCommandView` since you could simply use this view in another place other than inside a cell

Reply:
`ChatSuggestionsCommandView` -> isn't `ChatCommandSuggestionsView` more correct?

Reply:
Based on the comment above I would suggest `ChatInputSuggestionsCommandView` or `ChatCommandInputSuggestionsView`. Depends on the pattern that we chose.

Option 1:
`ChatInputSuggestionsViewController`
`ChatInputSuggestionsCommandView`
`ChatInputSuggestionsMentionView`

Option 2:
`ChatInputSuggestionsViewController`
`ChatCommandInputSuggestionsCommandView`
`ChatCommandInputSuggestionsMentionView`

I prefer Option 1, since it looks cleaner and easier to find because the prefix is always the same. WDYT?

Reply:
These views can be used in non-input contexts, so I don't find `Input` in the name correct. I liked `ChatCommandSuggestionsView` 2️⃣ 💵